docs/conf.py

```diff
@@ -0,0 +1,515 @@
+# pylint: disable=invalid-name
+"""
+xapi-db-load documentation build configuration file.
+
+This file is execfile()d with the current directory set to its
+containing dir.
+
+Note that not all possible configuration values are present in this
+autogenerated file.
+
+All configuration values have a default; values that are commented out
+serve to show the default.
+"""
+import os
+import re
+import sys
+from subprocess import check_call
+
+import edx_theme
```

Comment:
The cookiecutter is still not updated unfortunately, but you should drop usage of the edx-sphinx-theme using the steps described here: https://docs.openedx.org/en/latest/developers/how-tos/switch-to-the-sphinx-book-theme.html

You can do that as a follow-up if you want.